src/main/scala/com/github/tkawachi/doctest/DoctestPlugin.scala

```diff
@@ -85,10 +69,10 @@ object DoctestPlugin extends AutoPlugin {
    */
   val doctestGenSettings = Seq(
     doctestTestFramework := (doctestTestFramework ?? ScalaCheck).value,
-    doctestWithDependencies := (doctestWithDependencies ?? true).value,
     doctestDecodeHtmlEntities := (doctestDecodeHtmlEntities ?? false).value,
     doctestMarkdownEnabled := (doctestMarkdownEnabled ?? false).value,
     doctestMarkdownPathFinder := baseDirectory.value * "*.md",
+    testFrameworks            += new TestFramework("utest.runner.Framework"),
```

Comment:
So, for convenience, I've decided to add uTest to the list of supported test frameworks SBT has, so the application developer does not need to remember to add it in case he/she prefers uTest but is also using sbt-doctest.